R/binarise.R

```diff
@@ -62,13 +62,29 @@ gb_recode <- function(ValueTable, oldvariable, newvariable, func) {
 #' Makes multi-state Grambank-features binary in the appropriate manner.
 #'
 #' @param ValueTable data frame of the ValueTable from grambank-cldf (long).
-#' @param drop_multistate If TRUE,the multistate parent features of the binarised features are kept. If FALSE, they are dropped from the resulting data-frame.
-#' @param keep_raw_binary If TRUE and if the value table already contains some binarised features, they are kept. If false, they are overriden and replaced by values derived from the multistate features.
+#' @param drop_multistate logical vector. If TRUE,the multistate parent features of the binarised features are kept alongside the binarised features. If FALSE, they are dropped from the resulting data-frame, only binary and binarised features remain
```

Comment:
Something seems wrong here:

> If [drop_multistate is] TRUE,the multistate parent features of the binarised features are **kept** (?)

Shouldn't it be the other way round? Or is the argument name just misleading?

Reply:
yes, you're right. Can't believe I didn't catch that at first pass before.

Reply:
updated